src/test/java/core/basesyntax/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.model;
```

Comment:
models do not require separate tests usually
constructors/getters/setter are naturally involved into other tests